README.md

```diff
@@ -92,6 +93,36 @@ The above output includes only software that is visible in the container (i.e.,
 syft <image> --scope all-layers
 ```
 
+#### Format conversion (experimental)
+
+The ability to convert existing SBOMs means you can create SBOMs in different formats quickly, without the need to regenerate the SBOM from scratch, which may take significantly more time.
+
+```
+syft convert <ORIGINAL-SBOM-FILE> -o <NEW-SBOM-FORMAT>[=<NEW-SBOM-FILE>]
+```
+
+This feature is experimental and data might be lost when converting formats. Packages are the main SBOM component easily transferable across formats, while files and relationships are more likely to lose data. 
```

Comment:
> while files and relationships are more likely to lose data.
Maybe:
whereas files and relationships, as well as other information Syft doesn't support is more likely to be lost.